SECURITY.md

```diff
@@ -1,45 +1,139 @@
+bugs.
+
+### Encryption is not the solution to all the problems, but it helps
+
+Some data are encrypted before being saved in CouchDB (passwords for the
+accounts for example). Encrypting everything has some downsides:
+
+-   It's not possible to index encryped documents or do computations on the
+    encrypted fields in reasonable time
+    ([homomorphic encryption](https://en.wikipedia.org/wiki/Homomorphic_encryption)
+    is still an open subject).
+-   Having more encrypted data can globally weaken the encryption, if it's not
+    handled properly.
+-   If the encryption key is lost or a bug happen, the data is lost with no way
+    to recover them.
+
+So, we are more confortable to encrypt only some fields. And later, when we will
+have more experience and feedbacks from the user, extend the encryption to more
+fields.
+
+We are also working with [SMIS](https://project.inria.fr/smis/), a research lab,
+to find a way to securely store and backup the encryption keys.
+
+### Use standards (https, OAuth2)
+
+Standards like https and OAuth2 had a lot of eyes to look at them, and are
+therefore more robust. Reinventing the wheel can be valuable for some things.
+But for security, the cost is very high and only some very particular
+constraints can justify such an high cost. Cozy don't have these constraints, so
+we will stick to the standards.
+
+### Have a reliable way to deploy, with sane defaults
+
+It's important that deploying a cozy is well documented, doesn't require too
+many steps and can be automatized. An error on the installation can have
+dramatic effects, like the database being leaked on internet. So, we need to
+really take care of the devops experience. In particular, having sane defaults
+for the configuration will help to minimize the number of things he/she has to
+do, and such the number of places where he/she can make faux pas.
+
+### No one is perfect, code must be reviewed
+
+No code is directly pushed to the master branch in git. It has to be reviewed by
+at least one member of the core team (ideally the whole team), and this person
+can't be the author of the change. Even the best developers can make mistakes,
+and we don't pretend to be them. Our force is to work as team.
+
+### Be open to external contributors
+
+Our code is Open Source, external contributors can review it. If they (you?)
+find a weakness, please contact us by sending an email to security AT
+cozycloud.cc. This is a mailing-list specially setup for responsible disclosure
+of security weaknesses in Cozy. We will respond in less than 72 hours.
+
+When a security flaw is found, the process is the following:
+
+-   Make a pull-request to fix (on our private git instance) and test it.
+-   Deploy the fix on cozycloud.cc
+-   Publish a new version, announce it on
+    [the forum](https://forum.cozy.io/c/latest-information-about-cozy-security)
+    as a security update and on the mailing-lists.
+-   15 days later, add the details on the forum.
```

Comment:
I don't think we should keep this file. It seems to come from the stack documentation right? If yes,  we should just have a link to it. Not copying pasting the content. It's hard to maintain this way.

Reply:
New proposal :
- https://github.com/cozy/cozy-pass-web/blob/95bab48a55783d5df1a7b6d851511d59742dde15/SECURITY.md
- https://github.com/cozy/cozy-pass-web/blob/95bab48a55783d5df1a7b6d851511d59742dde15/CONTRIBUTING.md